v2/emailpassword/common-customizations/sessions/session-verification-in-api/get-session.mdx

```diff
@@ -174,6 +174,27 @@ export default async function likeComment(req: SessionRequest, res: any) {
 }
 ```
 
+</TabItem>
+<TabItem value="nextjs-app">
+
```

Comment:
this has become the same as the veirfySession docs. Maybe it's best if we actually call getSession with preparsedrequest and collecting response.. the whole thing basically.

Reply:
We can have a tip saying that you should use withSession instead (and link to the verifySession docs) if you don't want to do custom error handling since the main reason people want to use getSession instead of verifySession is for custom error handling.

Reply:
Applies to all snippets in this page

Reply:
Updated the `getSession` examples to use `withPreParsedRequestResponse`